libraries/core-react/src/Popover/PopoverAnchor.tsx

```diff
@@ -12,4 +12,4 @@ export const PopoverAnchor = forwardRef<HTMLDivElement, Props>(
   },
 )
 
-PopoverAnchor.displayName = 'eds-popover-anchor'
+// PopoverAnchor.displayName = 'eds-popover-anchor'
```

Comment:
Remove comment, this `displayName` is needed in `Popover.tsx`

Reply:
The generated displayName will be `PopoverAnchor` so I changed the checking to `popoverChildren.type.displayName === 'PopoverAnchor'` in `Popover.tsx`.